gpii/node_modules/flowManager/test/BrowserChannelTests.js

```diff
@@ -30,25 +30,35 @@ gpii.tests.flowManager.browserChannel.chromeUserToken = "chrome2";
 gpii.tests.flowManager.browserChannel.firefoxUserToken = "firefox";
 
 gpii.tests.flowManager.browserChannel.payloads = {
-    expectedChromeSettings: {
-        "screenReaderTTSEnabled": false,
-        "highContrastEnabled": true,
-        "invertColours": false,
-        "magnifierEnabled": true,
-        "magnification": 2,
-        "fontSize": "medium",
-        "simplifier": false,
-        "highContrastTheme": "white-black"
+    "org.chrome.cloud4chrome": {
+        settings: {
+            "screenReaderTTSEnabled": false,
+            "highContrastEnabled": true,
+            "invertColours": false,
+            "magnifierEnabled": true,
+            "magnification": 2,
+            "fontSize": "medium",
+            "simplifier": false,
+            "highContrastTheme": "white-black"
+        },
+        options: {
+            "path": "org.chrome.cloud4chrome"
+        }
     },
-    expectedFirefoxSettings: {
-        "screenReaderTTSEnabled": false,
-        "highContrastEnabled": true,
-        "invertColours": false,
-        "magnifierEnabled": true,
-        "magnification": 2,
-        "fontSize": "medium",
-        "simplifier": false,
-        "highContrastTheme": "black-white"
+    "org.mozilla.cloud4firefox": {
+        settings: {
+            "screenReaderTTSEnabled": false,
+            "highContrastEnabled": true,
+            "invertColours": false,
+            "magnifierEnabled": true,
+            "magnification": 2,
+            "fontSize": "medium",
+            "simplifier": false,
+            "highContrastTheme": "black-white"
+        },
+        options: {
+            "path": "org.mozilla.cloud4firefox"
```

Comment:
Hi @javihernandez - are these "path" directives needed (with a solution ID) - or are they just examples to be able to test the "options" block? If they're just examples, I suggest changing at least one of these to something different, such as: `"myopt": "some option for firefox"`, so it's clear that the options could be anything that the solution needs..